### PR TITLE
For https://github.com/cyrusimap/cyrus-sasl/issues/372 Fix segfault w…

### PR DESCRIPTION
**Pull request by quanah**
…ith GSSAPI mech

### CODE REVIEW

**Comment by ksmurchison** [comment]
Is the original code assuming a particular endianess which fails on some machines?

**Comment by quanah** [comment]
This comes from https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=629589

**Comment by quanah** [comment]
Which in github, refers back to #372 and #340

File: plugins/gssapi.c [hunk 1]
```diff
@@ -399,6 +399,7 @@ sasl_gss_encode(void *context, const struct iovec *invec, unsigned numiov,
     }
     
     if (output_token->value && output) {
+	int len;
```
Comment (aamelnikov) [inline plugins/gssapi.c]: I think this be uint32_t, int is really icky here. I think int can still be 2 bytes on some platforms.

**Pull request merged**